githubdelivery/src/main/java/io/dockstore/githubdelivery/GithubDeliveryCommandLineArgs.java

```diff
@@ -22,14 +22,54 @@ public File getConfig() {
         return config;
     }
 
-    @Parameters(commandNames = { "download-event" }, commandDescription = "Download github event from S3 bucket using key.")
-    public static class DownloadEventCommand {
+    @Parameters(commandNames = { "submit-event" }, commandDescription = "Submit a github event from S3 bucket using its key to the webservice.")
+    public static class SubmitEventCommand {
 
-        @Parameter(names = {"-k", "--key"}, description = "The key of the event in bucket. Format should be YYYY-MM-DD/deliveryid")
+        @Parameter(names = {"-c", "--config"}, description = "The config file path.")
+        private File config = new File("./" + DEFAULT_CONFIG_FILE_NAME);
+
+        @Parameter(names = {"-k", "--key"}, description = "The key of the event in bucket. Format should be YYYY-MM-DD/HH/deliveryid")
         private String bucketKey;
 
+        public File getConfig() {
+            return config;
+        }
         public String getBucketKey() {
             return bucketKey;
         }
     }
+    @Parameters(commandNames = { "submit-all" }, commandDescription = "Submit all github events from S3 bucket from a specific date to the webservice.")
+    public static class SubmitAllEventsCommand {
+
+        @Parameter(names = {"-c", "--config"}, description = "The config file path.")
+        private File config = new File("./" + DEFAULT_CONFIG_FILE_NAME);
+
+        @Parameter(names = {"-d", "--date"}, description = "All events from the date. Format should be YYYY-MM-DD")
+        private String date;
+
+        public File getConfig() {
+            return config;
+        }
+
+        public String getDate() {
+            return date;
+        }
+    }
+    @Parameters(commandNames = { "submit-hourly" }, commandDescription = "Submit all github events from S3 bucket from a specific date and hour to the webservice.")
```

Comment:
Nit, I'd probably call it `submit-hour` -- `submit-hourly` implies to me that will be doing a submit every hour.